lib/charms/operator_libs_linux/v0/juju_systemd_notices.py

```diff
@@ -462,13 +483,19 @@ def _main():
     """
     parser = argparse.ArgumentParser()
     parser.add_argument("-d", "--debug", action="store_true")
-    parser.add_argument("unit", type=str)
+    parser.add_argument("--unit", type=str)
+    parser.add_argument("services", nargs="*")
     args = parser.parse_args()
 
     # Intentionally set as global.
     global _juju_unit
     _juju_unit = args.unit
 
+    global _observed_services
```

Comment:
Again, `global` not required. You're not assigning the name as you're only using it via `_observed_services[service] = alias`. You'd need it if you were assigning the name itself, like  `_observed_services = {}`.

Reply:
Dropped global reference in https://github.com/canonical/operator-libs-linux/pull/128/commits/e3ab7193bffcdc5b04efb6fcd37f4b6e960ea7fb